daprdocs/content/en/reference/components-reference/supported-bindings/openai.md

```diff
@@ -0,0 +1,236 @@
+---
+type: docs
+title: "Azure OpenAI binding spec"
+linkTitle: "Azure OpenAI"
+description: "Detailed documentation on the Azure OpenAI binding component"
+aliases:
+  - "/operations/components/setup-bindings/supported-bindings/openai/"
+---
+
+## Component format
+
+To setup an Azure OpenAI binding create a component of type `bindings.azure.openai`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+See [this](https://learn.microsoft.com/azure/cognitive-services/openai/overview/) for the documentation for Azure OpenAI Service.
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: bindings.azure.openai
+  version: v1
+  metadata:
+  - name: apiKey
+    value: "1234567890abcdef"
+  - name: endpoint # Required
+    value: "https://myopenai.openai.azure.com"
+  - name: deploymentId # Required
+    value: "my-model"
+```
+{{% alert title="Warning" color="warning" %}}
+The above example uses apiKey as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Binding support | Details | Example |
+|--------------------|:--------:|--------|---------|---------|
+| `endpoint` | Y | Output | Azure OpenAI service endpoint URL. | `"https://myopenai.openai.azure.com"` |
+| `apiKey` | Y* | Output | The access key of the Azure OpenAI service. Only required when not using Azure AD authentication. | `"1234567890abcdef"` |
+| `deploymentId` | Y | Output | The name of the model deployment. | `"my-model"` |
+
```

Comment:
Can you also include the Azure AAD authentication fields into this table, as shown in the example below.  You can take the same descriptions from here https://docs.dapr.io/reference/components-reference/supported-bindings/eventgrid/

Reply:
Done. Please have a look.